src/haka_buddy/util.clj

```diff
@@ -20,5 +20,8 @@
                ;;Hax to fix tomcat double utf-8 encoding problem
                (new String (.getBytes val "ISO-8859-1") "UTF-8")]))))
 
-(defn get-attributes [request env & {:keys [names] :or {names shibbo-attribs}}]
-  (get-ajp-attributes names request))
+(defn get-attributes [request names]
+  (let [names (if names
+                names
+                shibbo-attribs)]
+    (get-ajp-attributes names request)))
```

Comment:
Maybe reorder `get-ajp-attributes` parameters to match `get-attributes` parameter order?